### PR TITLE
Switch to ansible-core-2.13 on CS8

### PR DESCRIPTION
**Pull request by mwperina**
Related to https://github.com/oVirt/ovirt-ansible-collection/pull/632 and https://github.com/oVirt/ovirt-engine/pull/754

Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: ovirt-hosted-engine-setup.spec.in [hunk 1]
```diff
@@ -72,14 +72,9 @@ Requires:       sudo
 Requires:       libvirt-client >= 6.6.0-9
 Requires:       virt-install
 
-Requires:       ansible-core >= 2.12
-# ansible-core 2.13 uses Python 3.9, and many modules are missing a Python 3.9
-# build on el8. TODO: Provide them? Instead of or in addition to Python 3.8?
-%if 0%{?rhel} < 9
-Conflicts:      ansible-core >= 2.13
-%endif
+Requires:       ansible-core >= 2.13
```
Comment (sandrobonazzola) [inline ovirt-hosted-engine-setup.spec.in:75]: I would consider adding a conflict on >=2.14 as per issue #68

Comment (mwperina) [reply ovirt-hosted-engine-setup.spec.in:75]: No, we need to be compatible with 2.14 unless we want to support Rocky/Alma 8.8 and 9.2.
The good thing is that both 2.13 and 2.14 are using Python 3.9, so hopefully not big changes needed from us

Comment (mwperina) [reply ovirt-hosted-engine-setup.spec.in:75]: Unless I'm missing something https://github.com/oVirt/ovirt-hosted-engine-setup/issues/68 should be fixed by https://github.com/oVirt/ovirt-hosted-engine-setup/pull/66

Comment (sandrobonazzola) [reply ovirt-hosted-engine-setup.spec.in:75]: great, thanks

**Review by sandrobonazzola — APPROVED** [review]

**Comment by mwperina** [comment]
Verified locally, host has been added to standalone engine successfully, ansible-core-2.13.5 has been installed on both engine and host

**Pull request merged**